tests/Config/TempSshConfigFileTest.php

```diff
@@ -23,25 +24,24 @@ public function setUp(): void
     public function testConstruct(): void
     {
         $file = new TempSshConfigFile();
-        $this->assertSame('', implode('', iterator_to_array($file)));
+        $this->assertSame('', CommandGeneratorTest::implodeTempFile($file));
         $this->assertFileExists(__DIR__ . '/../fixtures/.phpsu/config/ssh_config');
     }
 
     public function testConstructDifferentFolder(): void
     {
-        $this->markTestSkipped('Skipped because it does only work if user is not root');
-//        $reflection = new \ReflectionClass(TempSshConfigFile::class);
-//        $property = $reflection->getProperty('fileName');
-//        $property->setAccessible(true);
-//        $oldValue = $property->getValue();
-//        $property->setValue('/root/.phpsu/ssh_config');
-//        static::expectException(\Exception::class);
-//        static::expectExceptionMessage(sprintf('Directory "%s" was not created', '/root/.phpsu'));
-//        try {
-//            new TempSshConfigFile();
-//        } finally {
-//            $property->setValue($oldValue);
-//        }
+        $reflection = new \ReflectionClass(TempSshConfigFile::class);
+        $property = $reflection->getProperty('fileName');
+        $property->setAccessible(true);
+        $oldValue = $property->getValue();
+        $property->setValue('/etc/hosts/.phpsu/ssh_config');
```

Comment:
why specifically this path?

Reply:
this is always a file and never a directory. => will always result in an error